docs/excel/excel-add-ins-error-handling.md

```diff
@@ -63,8 +66,12 @@ The following table is a list of errors that the API may return.
 |`InvalidSelection`|The current selection is invalid for this operation.|
 |`ItemAlreadyExists`|The resource being created already exists.|
 |`ItemNotFound` |The requested resource doesn't exist.|
-|`NotImplemented`  |The requested feature isn't implemented.|
+|`nonBlankCellOffSheet`|Can't insert new cells because it would push non-empty cells off the end of the worksheet. These cells might appear empty but have blank values, some formatting, or a formula. Delete enough rows or columns to make room for what you want to insert and then try again.|
+|`NotImplemented`|The requested feature isn't implemented.|
+|`RangeExceedsLimit`|The cell count in range has exceeded the maximum supported number.|
 |`RequestAborted`|The request was aborted during run time.|
+|`RequestPayloadSizeLimitExceeded`|The request payload size has exceeded the limit. See the [Resource limits and performance optimization for Office Add-ins](../concepts/resource-limits-and-performance-optimization.md#excel-add-ins) article for more information. <br><br>This error occurs in Excel on the web.|
```

Comment:
Is it important to highlight this error *only* occurs in Excel on the web? Or is it implied heavily enough the way you have it?
```suggestion
|`RequestPayloadSizeLimitExceeded`|The request payload size has exceeded the limit. See the [Resource limits and performance optimization for Office Add-ins](../concepts/resource-limits-and-performance-optimization.md#excel-add-ins) article for more information. <br><br>This error only occurs in Excel on the web.|
```